### ISSUE TITLE
NTP Blocked on some networks

### ISSUE THREAD

**Issue by mcknight89** [adafruit/Adafruit-WebIDE#122]
Hey,

I was using my PI at my university and they have blocked NTP. so I get errors trying to login. 

I'm thinking a lot of company's / uni's will be the same.

Anyway I found away around it but using script I wrote that grabs the time from a remote server and sets the date on the pi there by fixing issues with webide.

I figured it could be useful if something like this was included in webide or at least some kind of time check to ensure the system time isn't something silly. I'm hoping this might help some people. 

<- Python Script ->

import urllib2
import os
url="http://time.mknetworks.co.uk"
req = urllib2.Request(url)
response = urllib2.urlopen(req)
os.system("date -s \""+response.read().rstrip('\n')+"\"")

**Comment by jwcooper** [comment]
We're closing all existing issues as we have a fairly large update to the webide. Please create a new issue for any issues found once we release the new version. Thanks!
